daemon/authorization/get_new_release.py

```diff
@@ -0,0 +1,45 @@
+# use token to get new release list
+import urllib.request
+import json
+
+class Get_new_release:
+    
+    new_releases_json = []
+    def __init__(self,access_token):
+        """
+            Get a list of new album releases featured in Spotify (shown, for example, on a Spotify player’s “Browse” tab).
+        """
+        self.headers = {
+            "Authorization": "Bearer " + access_token
+        } 
+# }
+    def get_new_list(self,count):
```

Comment:
I think you should add this function into the interface of MusicVideoInfoProvider and implement this function in SpotifyProvider.